src/main/scala/no/ndla/imageapi/service/ConverterService.scala

```diff
@@ -59,6 +60,52 @@ trait ConverterService {
         domainImageMetaInformation.captions.map(asApiCaption))
     }
 
+    def asApiImageMetaInformationWithSingleLanguage(domainImageMetaInformation: domain.ImageMetaInformation,
```

Comment:
For de andre konverteringsfunksjonene har vi mulighet til å velge mellom `...WithDomainUrl` og `...WithApplicationUrl`. Forkjellen mellom dem er hvor selve URLen til bildet blir hentet fra - enten en hardkodet streng (DomainUrl), eller hostnavnet fra http requesten (ApplicationUrl). Jeg tror det vil være lurt å ha samme logikk her også